lib/components/viewers/pattern-row.tsx

```diff
@@ -73,31 +70,36 @@ class PatternRow extends Component<Props, State> {
 
     const routeName = route.shortName ? route.shortName : route.longName
     const routeColor = getRouteColorBasedOnSettings(route.operator, route)
+    let fontSize = '32px'
+    if (routeName) {
+      if (routeName?.length >= 4 && routeName?.length <= 6) {
+        fontSize = '20px'
+      } else if (routeName?.length > 7) {
+        fontSize = '16px'
+      }
+    }
```

Comment:
Can this be replaced with `routeNameFontSize`?

Reply:
Yes good catch!!